docker/journald/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM docker.io/logdna/build-images:rust-buster-stable
-RUN apt update -y && apt install -y systemd
+RUN apt update -y && apt install -y systemd lsof procps
```

Comment:
We should add these to the base image going forward

Reply:
oh, TIL about https://github.com/logdna/build-images

Good call, I'll move it there as part of the follow up ticket.